jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/deploy/DeployServerlessApplicationForm.java

```diff
@@ -0,0 +1,104 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.lambda.deploy;
+
+import com.intellij.execution.util.EnvVariablesTable;
+import com.intellij.execution.util.EnvironmentVariable;
+import com.intellij.openapi.ui.ComboBox;
+import com.intellij.ui.AnActionButton;
+import com.intellij.ui.CommonActionsPanel;
+import com.intellij.ui.ToolbarDecorator;
+import com.intellij.ui.components.panels.Wrapper;
+import com.intellij.util.ui.UIUtil;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import software.aws.toolkits.jetbrains.services.cloudformation.Parameter;
+import software.aws.toolkits.jetbrains.ui.RegionSelector;
+
+import javax.swing.*;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class DeployServerlessApplicationForm {
+    @NotNull
```

Comment:
Nit : single line for these, see the EditFunctionPanel

Reply:
Can do - I'll need to see how to change my autoformatting settings so that it doesn't do this on me.